ibus.service.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=ibus daemon
+Documentation=man:ibus-daemon(1)
+
+[Service]
+Type=dbus
+BusName=org.freedesktop.IBus
+Environment=IBUS_SOCK=%t/ibus.socket
```

Comment:
Thinking this again, %t may be better than %t/ibus.socket since I cannot find mkdir